lib/presentation/crowdactions/crowdaction_view/commitment_item.dart

```diff
@@ -37,43 +37,46 @@ class CommitmentItem extends StatelessWidget {
             ),
           ),
           const SizedBox(width: 20),
-          Container(
-            alignment: Alignment.topLeft,
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                SelectableText(label,
-                    style: const TextStyle(
-                      fontSize: 15,
-                      color: Color(0xFF585858),
-                      fontWeight: FontWeight.w700,
-                    )),
-                const SizedBox(height: 5),
-                SelectableText(
-                  description,
-                  style: CollactionTextStyles.body,
-                ),
-                const SizedBox(height: 5),
-                SelectableText.rich(TextSpan(
-                    text: "Tags: ",
-                    style: const TextStyle(
-                      fontSize: 15,
-                      color: Color(0xFF585858),
-                      fontWeight: FontWeight.w500,
-                    ),
-                    children: [
-                      TextSpan(
-                        text: tagsPresentationOutput,
-                        style: CollactionTextStyles.body,
-                      )
-                    ])),
-                const SizedBox(height: 5),
-                SelectableText(
-                  "Points: $points",
-                  style: CollactionTextStyles.body,
-                ),
-                const SizedBox(height: 10)
-              ],
+          Expanded(
+            child: Container(
+              alignment: Alignment.topLeft,
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  SelectableText(label,
+                      style: const TextStyle(
+                        fontSize: 15,
+                        color: Color(0xFF585858),
+                        fontWeight: FontWeight.w700,
+                      )),
+                  const SizedBox(height: 5),
+                  SelectableText(
+                    description,
+                    style: CollactionTextStyles.body,
+                    maxLines: 2,
```

Comment:
I think that `maxLines` property is creating a space between description and tags
![image](https://user-images.githubusercontent.com/49217428/232505827-b9c4e765-ef54-453e-a785-0ddaf0b478bb.png)

Lets remove that line